app/api/datastore/entities/v1/maat/capital_details.rb

```diff
@@ -14,6 +14,23 @@ class CapitalDetails < Grape::Entity
           expose :national_savings_certificates, expose_nil: false
           expose :investments, expose_nil: false
           expose :properties, expose_nil: false
+
+          private
+
+          # TODO: Consider converting `properties` into a Grape::Entity like `payment`
```

Comment:
This is looking great @mseedat-moj ! I definitely thing it is worthing breaking these down into entities as it will help some of the indirection in Transformer::PROVIDER_DETAILS_RULES. Happy to help / pair.